kibble/config/config.go

```diff
@@ -20,10 +20,12 @@ import (
 	"io/ioutil"
 	"os"
 	"path"
+	"strings"
 
-	goversion "github.com/hashicorp/go-version"
 	"kibble/models"
 	version "kibble/version"
+
+	goversion "github.com/hashicorp/go-version"
```

Comment:
Doesnt look like this is used.

Reply:
https://github.com/shift72/kibble/pull/73/files/f55adf47870ef5d2f314a645bb34c69881664f0b#diff-409f8210b899b2dd8a50ccc1aac0c2b28eef55af0677b83f15a037216f019bdbR34

Reply:
Go format just gave the imports a little resort